src/renderer/api/json-api.ts

```diff
@@ -119,48 +117,53 @@ export class JsonApi<D = JsonApiData, P extends JsonApiParams = JsonApiParams> {
       reqInit,
     };
 
-    return cancelableFetch(reqUrl, reqInit).then(res => {
-      return this.parseResponse<D>(res, infoLog);
-    });
+    const res = await fetch(reqUrl, reqInit);
+
+    return this.parseResponse<D>(res, infoLog);
   }
 
-  protected parseResponse<D>(res: Response, log: JsonApiLog): Promise<D> {
+  protected async parseResponse<D>(res: Response, log: JsonApiLog): Promise<D> {
     const { status } = res;
 
-    return res.text().then(text => {
-      let data;
-
-      try {
-        data = text ? JSON.parse(text) : ""; // DELETE-requests might not have response-body
-      } catch (e) {
-        data = text;
-      }
-
-      if (status >= 200 && status < 300) {
-        this.onData.emit(data, res);
-        this.writeLog({ ...log, data });
-
-        return data;
-      } else if (log.method === "GET" && res.status === 403) {
-        this.writeLog({ ...log, data });
-      } else {
-        const error = new JsonApiErrorParsed(data, this.parseError(data, res));
-
-        this.onError.emit(error, res);
-        this.writeLog({ ...log, error });
-        throw error;
-      }
-    });
+    const text = await res.text();
+    let data;
+
+    try {
+      data = text ? JSON.parse(text) : ""; // DELETE-requests might not have response-body
+    } catch (e) {
+      data = text;
+    }
+
+    if (status >= 200 && status < 300) {
+      this.onData.emit(data, res);
+      this.writeLog({ ...log, data });
+
+      return data;
+    }
+
+    if (log.method === "GET" && res.status === 403) {
+      this.writeLog({ ...log, error: data });
+      throw data;
```

Comment:
Why did we change the logic to throw?

Reply:
I changed to throw because that matches how the rest of the 4XX errors are handled and it is what the renderer is expecting. Without this the renderer thinks this call succeeds.

Reply:
But this change affects _all_ api calls. These messages were ignored on purpose -> if we throw these then user will see a notification every time api call is made where user does not have access to -> lot's of spam.

Reply:
I didn't find that was the case. Let me check again.

Reply:
Did we ever check this?

Reply:
I have, the only change is that the logs to the dev-tools console are now red instead of green. That is all. The reason for this is `fetch` doesn't throw on HTTP errors, only on things like passing it not a string.